src/main/java/core/basesyntax/model/zoo/Animal.java

```diff
@@ -10,6 +12,8 @@
 
 @Entity
 @Inheritance(strategy = InheritanceType.SINGLE_TABLE)
+@DiscriminatorColumn(name = "type",
```

Comment:
what about `@DiscriminatorValue` for the children?

Reply:
done...